manifests/00-cluster-role.yaml

```diff
@@ -27,13 +27,6 @@ rules:
   verbs:
   - "*"
 
-- apiGroups:
-  - policy
-  resources:
-  - poddisruptionbudgets
-  verbs:
-  - "*"
-
```

Comment:
Will there be some window where, once this is applied, existing operator process reconciliations could fail to apply the PDB due to RBAC denials? Even if so I'm not sure it matters